### ISSUE TITLE
How to ignore subfolders from sync?

### ISSUE THREAD

**Issue by ArturJo** [lukasz-wronski/vscode-ftp-sync#222]
Czesc Lukasz,
I just wanted to know how to ignore remote subfolders from sync. What should I write in the ftp-sync.json file?
Could please help?

**Comment by dubbha** [comment]
Yep, ignore is weird and not very well described. I just faced this situation that I can't figure out. I need to ignore "vendors" directory, because it only exists remotely and I don't want it removed on every sync, but I also need a file called "webpack/vendors.js" to be updated on change. How can I ignore the root-level directory exactly, but not everything else that happens to have "vendors" in its name?

**Comment by dubbha** [comment]
README says:
`ignore - An array of escaped regular expression strings specifying paths to ignore. If a path matches any of these regular expressions then it will not be included in the sync. Default values are "\\.git", "\\.vscode" and ".DS_Store".`
I don't really get even this one. Why do we use \\ for .git, but not for .DS_store?
